proposals/pr001-ecma402.md

```diff
@@ -0,0 +1,208 @@
+# Meta-proposal: Towards common ECMA 402 API surface for Rust
+
+[fmil@google.com](mailto:fmil@google.com)
+
+2020-05-21
+
+See [README.md](README.md) for the full index of proposals.
+
+
+## Motivation
+
+_Usable text internationalization for Rust, **today**._
+
+### Why now?
+
+* There is a feature gap in Unicode support for rust.
+
+  Implementing a standard API could be a way to bridge that gap quickly by
+  using libraries that are available today to implement it now.  Then as ICU4X
+  matures, implementation can be switched.
+
+* It is as good a time as any to design this API for rust.
+
+  It should not matter that ICU4X in particular has not yet defined its
+  ergonomic APIs. The ECMA-402  surface should not depend on what is convenient
+  any specific implementing library, but rather should be ergonomic to use.
+
+## Goal
+
+_Validate the concept of a common ECMA 402 API, or fail quickly._
+
+This doc is submitted for comments as a sanity check on the approach.  Question
+to you, the potential commenter: **do you think this exercise is worth doing?**
+
+
+## Landscape
+
+Rust's support for internationalized text handling is "[still very immature and
+unstable, with most libraries being only weeks old and not used in production
+yet](https://www.arewewebyet.org/topics/i18n/)".  The [ECMA-402 support feature
+request for Rust](https://github.com/rust-lang/rfcs/issues/858) has been open
```

Comment:
I've replied to this issue and pointed them to icu4x repo. Thanks for finding it (2018 is the last time it was updated).